asreviewcontrib/datatools/sample.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+from pathlib import Path
+
+import pandas as pd
+from asreview import ASReviewData
+from asreview.data.base import load_data
+
+
+def _check_suffix(input_file, output_file):
+    # Also raises ValueError on URLs that do not end with a file extension
+    suffixes = [Path(input_file).suffix, Path(output_file).suffix]
+
+    set_ris = {".txt", ".ris"}
+    set_tabular = {".csv", ".tab", ".tsv", ".xlsx"}
+    set_suffixes = set(suffixes)
+
+    if len(set(suffixes)) > 1:
+        if not (set_suffixes.issubset(set_ris) or set_suffixes.issubset(set_tabular)):
+            raise ValueError(
+                "• Several file types were given; The input and the output file" 
+                "should be of the same type. "
+            )
+
+
+def sample(output_path, input_path, nr_records):
+    _check_suffix(input_path, output_path)
+
+    df_input = load_data(input_path).df
+
+    # Check for presence of any variation of a year column
+    if "publication_year" not in df_input.columns:
```

Comment:
Is it an idea to make this configurable? I imagine that `year` would also be a common name for the column holding this data. Maybe have an optional argument `--year_column` with default value `publication_year`?